docs/README.md

```diff
@@ -24,43 +24,23 @@ explained below:
 ### Static credentials 
 
 Static credentials can be provided by adding an `node_ip`, `username` and `password` in-line in the
-Rubrik provider block:
-
-Usage:
-
-```hcl
-provider "rubrik" {
-  node_ip     = "10.255.41.201"
-  username    = "admin"
-  password    = "RubrikTFDemo2019"
-}
-```
+Rubrik provider block.
+s
```

Comment:
I've reverted this change. The intent was to remove the examples due to security checks. Upon review the examples can stay since they are not in code.